wandb/apis/public.py

```diff
@@ -4980,6 +4980,7 @@ def __init__(self, api: Api, name, path: str = None) -> None:
         with open(os.path.join(self._fpath, "wandb-job.json")) as f:
             self._source_info = json.load(f)
         self._entrypoint = self._source_info.get("source", {}).get("entrypoint")
+        self._args = self._source_info.get("source", {}).get("args")
```

Comment:
I feel like `_args` is generic to the point of potentially being confusing. However I do not have a better name :(

Reply:
Valid, I'm tempted to leave as is since it is identical to the settings from where the value is sourced